src/libraries/System.Security.Cryptography.Algorithms/src/System.Security.Cryptography.Algorithms.csproj

```
@@ -570,6 +570,9 @@
     <Compile Include="System\Security\Cryptography\AesCcm.Unix.cs" />
     <Compile Include="System\Security\Cryptography\AesGcm.Unix.cs" />
   </ItemGroup>
+  <ItemGroup Condition=" '$(TargetsBrowser)' == 'true'">
+    <Compile Include="Internal\Cryptography\RandomNumberGeneratorImplementation.WebAssembly.cs" />
```

nit: maybe we should call this .Browser.cs to be consistent:

```suggestion
    <Compile Include="Internal\Cryptography\RandomNumberGeneratorImplementation.Browser.cs" />
```